protocol/lib/eth/util_test.go

```diff
@@ -9,10 +10,104 @@ import (
 	libeth "github.com/dydxprotocol/v4-chain/protocol/lib/eth"
 	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
 	bridgetypes "github.com/dydxprotocol/v4-chain/protocol/x/bridge/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
 	ethcoretypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/stretchr/testify/require"
 )
 
+func TestGetBridgeEventAbi(t *testing.T) {
+	results := make([]*abi.ABI, 0)
+	mu := &sync.Mutex{}
+	wg := new(sync.WaitGroup)
```

Comment:
nit: consider using a non-pointer version of the waitgroup so it is allocated on the stack.
```suggestion
	wg := sync.WaitGroup{}
```
or `var wg sync.WaitGroup{}`